hack/testing/check-EFK-running.sh

```diff
@@ -1,352 +1,57 @@
-  checkKibanaStarted "$pod" || EXIT_CODE=1
-done
-for pod in $(oc get pods -l component=kibana-ops -o name); do
-  checkKibanaStarted "$pod" || EXIT_CODE=1
-done
-
-echo $TEST_DIVIDER
-echo "Checking if ES contains common data model index templates"
-for pod in $(oc get pods -l component=es -o jsonpath='{.items[*].metadata.name}'); do
-  checkESContainsIndexTemplates "$pod" || EXIT_CODE=1
-done
-if [[ "$CLUSTER" == "true" ]]; then
-  for pod in $(oc get pods -l component=es-ops -o jsonpath='{.items[*].metadata.name}'); do
-    checkESContainsIndexTemplates "$pod" || EXIT_CODE=1
-  done
-fi
-
-echo $TEST_DIVIDER
-exit $EXIT_CODE
+set -o errexit
+set -o nounset
+set -o pipefail
+if [[ "${VERBOSE:-}" = "true" ]]; then
+	set -o xtrace
+fi
+
+OAL_EXPECTED_DEPLOYMENTCONFIGS=( "logging-kibana" "logging-curator" )
+OAL_EXPECTED_ROUTES=( "kibana" )
+OAL_EXPECTED_SERVICES=( "logging-es" "logging-es-cluster" "logging-kibana" )
+OAL_EXPECTED_OAUTHCLIENTS=( "kibana-proxy" )
+OAL_EXPECTED_DAEMONSETS=( "logging-fluentd" )
+OAL_ELASTICSEACH_COMPONENTS=( "es" )
+OAL_KIBANA_COMPONENTS=( "kibana" )
+
+if [[ $# -eq 1 ]]; then
+	# There is an ops cluster set up, so we
+	# need to expect to see more objects.
+	OAL_EXPECTED_DEPLOYMENTCONFIGS+=( "logging-kibana-ops" "logging-curator-ops" )
+	OAL_EXPECTED_ROUTES+=( "kibana-ops" )
+	OAL_EXPECTED_SERVICES+=( "logging-es-ops" "logging-es-cluster-ops" "logging-kibana-ops" )
+	OAL_ELASTICSEACH_COMPONENTS+=( "es-ops" )
+	OAL_KIBANA_COMPONENTS+=( "kibana-ops" )
+fi
+
+# Currently one DeploymentConfig per ElasticSearch
+# replica is created, and is therefore given a long
+# unique name that we do not know beforehand. We
+# only know that there should be DCs with the
+# logging-es- prefix, so we cheat now to look it up
+# and keep the cluster rollout test clean.
+# TODO: This will not be necessary when StatefulSets
+# are used to deploy the cluster instead.
+es_dcs="$( oc get deploymentconfigs --selector component=elasticsearch -o jsonpath='{.items[*].metadata.name}' | grep -E "^logging-es-[a-zA-Z0-9]{8}" )"
+if [[ "$( wc -w <<<"${es_dcs}" )" -ne 1 ]]; then
+	os::log::fatal "Expected to find one ElasticSearch DeploymentConfig, got: '${es_dcs:-"<none>"}'"
+fi
+OAL_EXPECTED_DEPLOYMENTCONFIGS+=( ${es_dcs} )
+if [[ $# -eq 1 ]]; then
+	es_ops_dcs="$( oc get deploymentconfigs --selector component=elasticsearch -o jsonpath='{.items[*].metadata.name}' | grep -E "^logging-es-ops-[a-zA-Z0-9]{8}" )"
+	if [[ "$( wc -w <<<"${es_ops_dcs}" )" -ne 1 ]]; then
+		os::log::fatal "Expected to find one OPS ElasticSearch DeploymentConfig, got: '${es_ops_dcs:-"<none>"}'"
+	fi
+	OAL_EXPECTED_DEPLOYMENTCONFIGS+=( ${es_ops_dcs} )
+fi
+
+export OAL_EXPECTED_DEPLOYMENTCONFIGS
+export OAL_EXPECTED_ROUTES
+export OAL_EXPECTED_SERVICES
+export OAL_EXPECTED_OAUTHCLIENTS
+export OAL_EXPECTED_DAEMONSETS
+export OAL_ELASTICSEACH_COMPONENTS
+export OAL_KIBANA_COMPONENTS
+
+"${OS_O_A_L_DIR}/test/cluster/rollout.sh"
```

Comment:
`OS_O_A_L_DIR` is not defined.  You could `export` it from `hack/testing/logging.sh`